pkg/types/installconfig.go

```diff
@@ -77,6 +77,10 @@ type InstallConfig struct {
 	// If unset, the cluster will not be configured to use a proxy.
 	// +optional
 	Proxy *Proxy `json:"proxy,omitempty"`
+
+	// ImageContentSources is the lists of sources/repositories that can be used to pull the same content.
```

Comment:
Before we hit GA we should clean up this description, I think it's fine for now.